src/riak_kv_test_util.erl

```diff
@@ -326,6 +339,7 @@ start_app_and_deps(Application, Started) ->
         true ->
             {ok, Started};
         false ->
+            _Apps = application:which_applications(),
```

Comment:
I'm curious as to what this does

Reply:
I have no idea…I'll have to have a read of the whole file again. This is what happens with so much time between commit and review, eh?

Reply:
I think it just needs to be taken out, probably a different idea, and then underbarred the binding just to make it compile.

Reply:
Oh yeah, it was for some printf debugging